Travis refuses to compile ... again

Recent Travis builds seem to always fail recently. Seems, like its due to a recent change to adobe flex:
```
0.93s$ ant test-$BUILD

Buildfile: /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/build.xml

init:

    [mkdir] Created dir: /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/target

    [mkdir] Created dir: /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/target/report

test-build-release:

    [mxmlc] Loading configuration file /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/flex/frameworks/flex-config.xml

    [mxmlc] Apache Flex Compiler (mxmlc)

    [mxmlc] Version 4.16.0 build 20170305

    [mxmlc] Copyright 2017 The Apache Software Foundation.

    [mxmlc] 

    [mxmlc] /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/flex/frameworks/flex-config.xml(65): Error: unable to open '{playerglobalHome}/11.1/playerglobal.swc'

    [mxmlc] 

    [mxmlc]       </external-library-path>

    [mxmlc] 

BUILD FAILED

/home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/build.xml:88: The following error occurred while executing this line:

/home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/build.xml:65: mxmlc task failed
```

@brrritssocold or whoever has an idea how to fix this, could you look into that, please?

Link to a failing build: https://travis-ci.org/Kitteh6660/Corruption-of-Champions-Mod/jobs/280141233

Now it fails in a similar way:
```
$ ant test-$BUILD
Buildfile: /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/build.xml

init:
    [mkdir] Created dir: /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/target
    [mkdir] Created dir: /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/target/report

test-build-release:
    [mxmlc] Loading configuration file /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/flex/frameworks/flex-config.xml
    [mxmlc] Apache Flex Compiler (mxmlc)
    [mxmlc] Version 4.16.0 build 20170305
    [mxmlc] Copyright 2017 The Apache Software Foundation.
    [mxmlc] 
    [mxmlc] /home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/flex/frameworks/flex-config.xml(85): Error: unable to open 'libs/player/27.0'
    [mxmlc] 
    [mxmlc]       </library-path>
    [mxmlc] 

BUILD FAILED
/home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/build.xml:90: The following error occurred while executing this line:
/home/travis/build/Kitteh6660/Corruption-of-Champions-Mod/build.xml:67: mxmlc task failed

Total time: 0 seconds


The command "ant test-$BUILD" exited with 1.
```

Whats confusion to me is: setup-flex.sh sets the PLAYER_VERSION to 23.0, but Travis tries to open `libs/player/27.0`

\~reopening\~

--- a/devTools/ci/setup-flex.sh
+++ b/devTools/ci/setup-flex.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 VERSION='4.16.0'
-PLAYER_VERSION='23.0'
+PLAYER_VERSION='27.0'
 GLOBAL_PLAYER_DIR="frameworks/libs/player/$PLAYER_VERSION"
 GLOBAL_PLAYER_PATH="$GLOBAL_PLAYER_DIR/playerglobal.swc"
 
@@ -14,7 +14,7 @@ echo "Creating $GLOBAL_PLAYER_DIR"
 mkdir -vp $GLOBAL_PLAYER_DIR
 
 # TODO generate link with version number with sed or awk?
-wget -nc http://download.macromedia.com/get/flashplayer/updaters/23/playerglobal23_0.swc -O"$GLOBAL_PLAYER_PATH"
+wget -nc http://download.macromedia.com/get/flashplayer/updaters/27/playerglobal27_0.swc -O"$GLOBAL_PLAYER_PATH"
 
 # Setup flash player. You read that right! Flash player for linux is back!
 wget -N https://fpdownload.macromedia.com/pub/labs/flashruntimes/flashplayer/linux64/flash_player_sa_linux.x86_64.tar.gz

